php/functions.php

```diff
@@ -47,30 +42,126 @@ function addParagraphs(array $paragraphs) :string {
  * @param $addSubmit string inserts the string in the text area to the database and pairing it with an `id` and a `deleted` number
  */
 function addParagraphToDb (PDO $db, string $addSubmit) : void{
-    $query = $db->prepare("INSERT INTO `about_me` (`paragraph`) VALUES (:newParagraph);");
-    $query->bindParam(':newParagraph', $addSubmit);
-    $query->execute();
+    if (empty(trim($addSubmit))){
+        header('Location: admin_page.php');
+    } else if (strlen($addSubmit) < 1000) {
+        $query = $db->prepare("INSERT INTO `about_me` (`paragraph`) VALUES (:newParagraph);");
+        $query->bindParam(':newParagraph', $addSubmit);
+        $query->execute();
+    } else {
+        header('Location: admin_page.php');
```

Comment:
Again here, I wouldn't do these, i'd just return false or something else like that instead

Reply:
same as the one up